packages/bootstrap-4/test/__snapshots__/dateTimeWidget.test.tsx.snap

```diff
@@ -0,0 +1,29 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`DateTimeWidget simple 1`] = `
+<div
+  className="mb-0 form-group"
+>
+  <label
+    className="form-label"
+    htmlFor="_id"
+  >
+    Some simple label
+    *
+  </label>
+  <input
+    autoFocus={true}
+    className="form-control"
+    disabled={true}
+    id="_id"
+    multiple={false}
+    name="nodejs"
+    onBlur={[Function]}
+    onChange={[Function]}
+    onFocus={[Function]}
+    required={true}
+    type="color"
+    value="0NaN-NaN-NaNTNaN:NaN:NaN.NaN"
```

Comment:
You should probably pass a valid date value for this component, so that the test actually tests that the data rendered will be correct. This seems like an undesirable output 😄

Reply:
opz, I will change it.